app/react/components/Elements/FeatureToggle.tsx

```diff
@@ -21,11 +21,13 @@ FeatureToggle.defaultProps = {
 };
 
 function mapStateToProps({ settings }: any, ownProps: OwnPropTypes) {
-  const features = settings.collection.get('features') || Immutable.fromJS({});
+  const features = settings.collection.get('features');
+
   return {
-    featureActivated: features.get(ownProps.feature),
+    featureActivated: features ? Boolean(get(features.toJS(), ownProps.feature)) : false,
```

Comment:
Why is this change to lodash needed? Why not use `Boolean(features.get(ownProps.feature))`?  `toJS` operations tend to be expensive.

Reply:
We will avoid using toJS by using getIn.